parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoWriteSupport.java

```diff
@@ -318,7 +318,7 @@ final void writeRawValue(Object value) {
     @Override
     final void writeRawValue(Object value) {
       ByteString byteString = (ByteString) value;
-      Binary binary = Binary.fromByteArray(byteString.toByteArray());
+      Binary binary = Binary.fromConstantByteArray(byteString.toByteArray());
```

Comment:
From ByteString javadocs (https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/ByteString): ```Like String, the contents of a ByteString can never be observed to change, not even in the presence of a data race or incorrect API usage in the client code.```

So using `fromConstantByteArray` is correct here.